store/statistics.js

```diff
@@ -0,0 +1,31 @@
+import apiRequest from 'api'
+
+class Statistics {
+  constructor(rootStore) {
+    this.rootStore = rootStore
+  }
+
+  retrieveStatistics = async dataProviderId => {
+    try {
+      const { data } = await apiRequest(
+        `/data-providers/${dataProviderId}/statistics`,
```

Comment:
I was using it like this in my actions:

`/data-providers/${this.rootStore.dataProviderId}/statistics`

Reply:
Please, check the current solution. Does it satisfy you?

I try to make functions as pure as possible. Maybe here it's not the best what I could do. Let me know.